web/styling/positioning-tips/direction-column-reverse.md

```diff
@@ -57,7 +57,7 @@ ___
 ```
 The four child boxes inside class parent will be displayed **vertically** on top of one another in reverse order, starting from the bottom of the parent element:
 
-![566ed55387abab0c00bccab0.svg](https://img.enkipro.com/5cd4ebde7bebedb1168c64d4f3d8ee61.png)
+![566ed55387abab0c00bccab0.svg](https://img.enkipro.com/9335ed930eba8f366028beb7d356daf4.png)
```

Comment:
could we edit this image a bit more? I think there's a lot of unnecessary purple background to the right @Stefan-Stojanovic